### PR TITLE
bottou2010large_2+zayne-sprague: 1st pass of rough draft

### PR DESCRIPTION
**Pull request by Zayne-sprague**

### CODE REVIEW

File: _summaries/bottou2010large_2.md [hunk 1]
```diff
@@ -1,19 +1,126 @@
...
+$$
+
+The authors found that SGD is quicker to reach a predefined Excess Error
+
+With sufficient training samples, you can model the Expected Risk (the actual distribution) more accurately, and be able to approach some predefined value of Expected Risk quicker with SGD than GD can.
+
+
+_**Does 2nd Order Stochastic Gradient Decent (2SGD) have any benefits?**_
+
+2SGD, with sufficient assumptions (convexity and regularity), will approach optimum solution as quickly as possible
```
Comment (patelojas) [inline _summaries/bottou2010large_2.md:92]: On page 2 of the paper, 2GD is brought up in slight detail. I think it might be valuable to discuss it here in a sentence or two to compare to 2SGD

File: _summaries/bottou2010large_2.md [hunk 2]
```diff
@@ -1,19 +1,126 @@
...
+
+
+_**How well does the paper perform?**_
+
+SGD is used quite often now, so it did become quite popular
+
+The authors tested SGD, ASGD, and SGDQN on various types of datasets showing that Stochastic Gradient Decent is capable of the Expected Risk quickly beating out other algorithms (30s -> 2.3s)
+
+//TODO include plots here
+
```
Comment (patelojas) [inline _summaries/bottou2010large_2.md:108]: Including some information from 2.3 might be valuable here.

File: _summaries/bottou2010large_2.md [hunk 3]
```diff
@@ -1,19 +1,126 @@
...
+- Approximation Error (how many functions can you search over for the optimum)
+- Estimation Error (how well does the training set represent the real distribution)
+- Optimization Error (did you stop the optimization too early, or set $$p$$ too high)
+
+You can then choose, specific to your problem, which of these risks to minimize
+
+In some cases GD may be the better option than SGD if you are working on a smaller machine learning problem.
+
 
 ## TL;DR
```
Comment (patelojas) [inline _summaries/bottou2010large_2.md:122]: I think you did a good job of explaining the differences between GD and SGD without going too into mathematics. In section 3, the paper goes heavy into math and it might not necessarily belong in a summary.

Comment (Zayne-sprague) [reply _summaries/bottou2010large_2.md:122]: Yeah, I agree, the paper felt more like a very deep proof.  And as cool as that is, I don't think you have to really remember it (it's more like, okay this is the right conclusion to come to).

That being said though if you think I need to add any other equations totally let me know, I tried to only pull the important ones.

File: _summaries/bottou2010large_2.md (hunk 1 above)
Comment (Zayne-sprague) [reply _summaries/bottou2010large_2.md:98]: Totally, I forgot to add the equation for it and a quick explanation.  Good catch! I added the eq and very short sentence about how its different.

File: _summaries/bottou2010large_2.md (hunk 2 above)
Comment (Zayne-sprague) [reply _summaries/bottou2010large_2.md:118]: Good eye, I added a sentence about how they showed SGD is used for various ML tasks beyond Deep learning (SVM/Kmeans etc.)
